### PR TITLE
Optuna

### PR DESCRIPTION
**Pull request by theUnrealSamurai**
**Issue:**
**Alexa rank:**
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: icons/optuna.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...1.6 195c14.5 8.3 39.3 15 55.4 15 8.9 0 22.9-2 32-4.7 48.8-14 83.4-62.8 80.7-113.7-1.4-25.5-10.8-46.9-27.9-63.2-22-21.2-56-31.5-89.3-27.3zm40 16.8c27.1 7.5 49.5 26.1 60 49.8 6.7 14.9 8.4 35.7 4.5 52.2-6.5 26.8-26.2 51.1-52.5 64.8-27.9 14.4-61.2 15-87.9 1.7-20.3-10.2-37.7-28.8-46.8-50.2-4.3-9.8-5.8-16.7-6.5-28.7C11 64.1 42.5 27.6 91.5 17c12.8-2.8 39.3-2.3 51 .9z"/><path d="M96.3 29.5C60.8 35.9 31.6 61 24.5 91.4c-9 37.9 20.2 77.7 65.5 89.3 12.9 3.3 29.6 3.5 42.5.4 46.1-11 74.4-58.7 60.1-101.1-3.4-10.1-7.9-17.2-16.8-26.5-19.3-20.3-49.5-29.4-79.5-24zm31.3 10c23.8 4.5 44 22.8 51 46.1 2.4 7.9 2.4 24.5 0 33.4-2.3 8.8-8.6 21.5-13.5 27.3-13.3 15.9-33.2 24.8-56.1 25.1-13.8.1-19.4-1.1-31.5-6.9-15.5-7.4-27.8-20.8-33.6-36.5-2.1-5.6-2.4-8.3-2.4-19 0-10.5.4-13.6 2.3-19 3.9-10.9 9.8-19.8 19.7-29.7 8-7.9 10.5-9.8 19.6-14.2 16-7.7 28.7-9.6 44.5-6.6z"/><path d="M96.2 58C67.3 65.6 49 91.5 55.4 115.6c5.9 22.2 29.6 39.3 54.7 39.6 15.1.1 27.4-4.3 37.5-13.5 12.3-11.2 17.9-24.3 17.2-40-.3-8.2-.8-10.2-4.1-17.1C150.1 62.3 122.8 51 96.2 58zm34.3 16.4c7.3 3.3 12.8 8.4 16.3 15.1 2.2 4.2 2.7 6.3 2.7 13 0 6.9-.4 8.9-3.1 14.6-8.7 18.3-29.6 27.8-48.5 22-21.3-6.7-31.5-27.2-22.4-45.1 9.6-18.8 36-28.3 55-19.6z"/></svg>
```
Comment (dirien) [inline icons/optuna.svg:1]: Can you check the icon file:

best thing is o run `npm run svgo -- icons/optuna.svg` and then `npm run lint`


```
  x attr-1 0:0 Expected attribute 'role', didn't find it
                 At node <svg> (0:0)
  x attr-1 0:0 Found extra attributes ["version","width","height"] with whiteli
               sting enabled
                 At node <svg> (0:0)
  x icon-title No icon with title "" found in simple-icons.json
  x icon-size Size of <path> must be exactly 24 in one dimension; the size is c
              urrently 219.579 x 209.824
  x extraneous Unexpected character(s), most likely extraneous whitespace, dete
               cted in SVG markup
  x icon-centered <path> must be centered at (12, 12); the center is currently 
                  (110.058, 105.088)
  x elm Found 0 elements for 'svg > title', expected 1
  x elm 0:99 Found 3 elements for 'svg > path', expected 1
               At node <path> (0:99)
  x elm 0:561 Found 3 elements for 'svg > path', expected 1
                At node <path> (0:561)
  x elm 0:1041 Found 3 elements for 'svg > path', expected 1
                 At node <path> (0:1041)
  x elm 0:99 Element disallowed
               At node <path> (0:99)
  x elm 0:561 Element disallowed
                At node <path> (0:561)
  x elm 0:1041 Element disallowed
                 At node <path> (0:1041)
```

**Review by dirien — CHANGES_REQUESTED** [review]

**Comment by adamrusted** [comment]
Even after tidying up this SVG, it doesn't quite line up to the raster (see red in background for SVG)

image

**Comment by adamrusted** [comment]
@theUnrealSamurai - could you please close this PR, or update based on feedback? I'll close this in a few days if we don't hear anything.

**Comment by adamrusted** [comment]
Closing due to lack of activity.

**Pull request closed without merging**